### ISSUE TITLE
Qemu-ovs package should not depend on volumedriver-server

### ISSUE THREAD

**Issue by saelbrec** [openvstorage/qemu#2]
root@node:~/archive/qemu# dpkg --info qemu_2.3+dfsg-0ovs1.0-5ubuntu9.3_amd64.deb 
 new debian package, version 2.0.
 size 126582 bytes: control archive=1118 bytes.
    1403 bytes,    30 lines      control  
     330 bytes,     5 lines      md5sums  
 Package: qemu
 Version: 99:2.3+dfsg-0ovs1.0-5ubuntu9.3
 Architecture: amd64
 Maintainer: Ubuntu Developers ubuntu-devel-discuss@lists.ubuntu.com
 Installed-Size: 442
 Depends: qemu-system (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.3), qemu-user (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.3), qemu-utils (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.3), volumedriver-server (>= 5.7.0)
 Suggests: qemu-user-static

**Comment by redlicha** [comment]
Fixed on the support/volumedriver-6.0.x branch.

**Comment by kinvaris** [comment]
PASSED:

Version:

```
root@cmp01:~# apt-cache policy qemu
qemu:
  Installed: 99:2.3+dfsg-0ovs1.0-5ubuntu9.4
  Candidate: 99:2.3+dfsg-0ovs1.0-5ubuntu9.4
  Version table:
 *** 99:2.3+dfsg-0ovs1.0-5ubuntu9.4 0
        500 http://apt.openvstorage.org/ unstable/main amd64 Packages
        100 /var/lib/dpkg/status
     2.0.0+dfsg-2ubuntu1.27 0
        500 http://be.archive.ubuntu.com/ubuntu/ trusty-updates/universe amd64 Packages
        500 http://security.ubuntu.com/ubuntu/ trusty-security/universe amd64 Packages
     2.0.0~rc1+dfsg-0ubuntu3 0
        500 http://be.archive.ubuntu.com/ubuntu/ trusty/universe amd64 Packages
```

Dependancies of package `qemu` in `unstable`:

```
Package: qemu
Version: 99:2.3+dfsg-0ovs1.0-5ubuntu9.4
Architecture: amd64
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Installed-Size: 442
Depends: qemu-system (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4), qemu-user (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4), qemu-utils (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4)
```

Dependancies of package `qemu` in `fargo`:

```
Package: qemu
Version: 99:2.3+dfsg-0ovs1.0-5ubuntu9.4
Architecture: amd64
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Installed-Size: 442
Depends: qemu-system (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4), qemu-user (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4), qemu-utils (>= 99:2.3+dfsg-0ovs1.0-5ubuntu9.4)
```
